test/system/search_test.rb

```diff
@@ -8,15 +8,15 @@ class SearchSystemTest < ApplicationSystemTestCase
   end
 
   test "show search results" do
-    fill_in "search_input", with: "artist"
-    find(:test_id, "search_input").send_keys(:enter)
+    fill_in "query", with: "artist"
+    find("#search").send_keys(:enter)
```

Comment:
In system test, we should use test_id to find element, because use test_id we can reduce relation between test case and frontend code. The system test will be more maintainable